testsuite/model/src/test/java/org/keycloak/testsuite/model/parameters/Infinispan.java

```diff
@@ -40,6 +49,10 @@
       .add(CacheRealmProviderSpi.class)
       .add(CacheUserProviderSpi.class)
       .add(InfinispanConnectionSpi.class)
+      .add(StickySessionEncoderSpi.class)
+      .add(TimerSpi.class)
+      .add(UserSessionPersisterSpi.class)
+      .add(UserLoginFailureSpi.class)
```

Comment:
`UserLoginFailureSpi` and `TimerSpi` should be in default enabled `Spi`s in KeycloakModelTest.

Reply:
`TimerSpi` moved to `KeycloakModelTest`. `UserLoginFailureSpi` was already there.